README.md

```diff
@@ -301,11 +306,57 @@ updateNavigationBar({
     expandedTitle: 'Hello, World!',
     showBackButton: true,
     showReloadButton: false,
-    showProfileButton: false,
-    backgroundColor: '#FF0000', // red
+    backgroundColor: '#FF0000',
+    leftNavigationIcons: [
+        {
+            name: 'icon name',
+            iconEnum: 'SOME_ICON',
+            badge: {
+                show: true,
+                nativeLogic: 'INBOX',
+            },
+        },
+    ],
+    rightNavigationIcons: [
+        {
+            name: 'icon name',
+            iconEnum: 'icon enum value',
+            icon: {
+                url: 'https://path/to/icon',
+                urlDark: 'https://path/to/icon/dark',
+            },
+            badge: {
+                show: true,
+                number: 1,
+            },
+        },
+    ],
+    resetToDefaultState: true,
+    trackingProperties?: {'name': 'some icon clicked'},
 });
 ```
 
+### onNavigationBarIconClicked
+
+Listen to navigation bar icon clicks and execute a callback function
+
+Requires App versions 14.8 or higher
+
+#### React example
+
+```tsx
+React.useEffect(() => {
```

Comment:
wouldn't it make sense for this effect to be applied automatically when doing useSetNavbarState ? meaning from the outside you simply pass a certain "id" for the top icons with an "onPress" and it's subscribed / unsubscribed automatically?

That would also help encapsulate it for when (eventually) we want to use the same logic not inside app and the pattern of subscribe / unsubscribe makes no sense because we'll just render a Touchable.

Reply:
That could make sense. We could update webapp's `useSetNavbarState` after merging this.

This repo (webview-bridge) knows nothing about react, it just provides wrappers around the postMessage mechanism